seqr/views/apis/anvil_workspace_api.py

```diff
@@ -174,8 +178,16 @@ def create_project_from_workspace(request, namespace, name):
         logger.error('Uploading sample IDs to Google Storage failed. Errors: {}'.format(str(ee)), request.user,
                      detail=sample_ids)
 
-    # Send a slack message to the slack channel
-    _send_load_data_slack_msg(project, ids_path, data_path, request_json['sampleType'], request.user)
+    # use airflow api to trigger AnVIL dags
+    try:
+        _trigger_anvil_dags(project, data_path, request_json)
+        # Send a slack message to the slack channel
+        _send_load_data_slack_msg(project, ids_path, data_path, request_json['sampleType'], request.user)
+    except Exception as e:
+        logger.error(e)
+        safe_post_to_slack(SEQR_SLACK_ANVIL_DATA_LOADING_CHANNEL, str(e))
```

Comment:
So I discussed this with the loading team, and we've determined the following is the notification behavior we want if the DAG fails to trigger for any reason:
1) We should always send the "success" message (`_send_load_data_slack_msg`) regardless of whether or not the dag actually was triggered. This allows that channel to have a full record of all loading requests we receive, regardless of how the DAG ends up being triggered. You do not need to change the text of the message if the trigger does fail, it is okay to leave it as is with "DAG was triggered"
2) We want to send notifications of failure to the `seqr_loading_notifications` channel (you will need to add a new variable for that). This message needs to include the error that occurred, the dag name, and the dag variables. I think this will be a good template:
```
"""
ERROR triggering AnVIL loading for project {project_guid}: {e} 
  
DAG {dag_name} should be triggered with following: 
```{dag}```
"""
```
3) We should leave the `logger.error(e, request.user)` as it is